test/deprecated-keyword/handleConfigParam.set.good

```diff
@@ -0,0 +1,5 @@
+warning: don't use x, use y
+note: 'x' was set via a compilation flag
```

Comment:
Do we use this "compilation flag" term elsewhere?  My head goes to phrases like "compiler command-line option" or "compiler command-line flag" or "compiler option" or "compiler flag".  It looks like our man page uses "OPTIONS" when introducing them, but then "flags" when describing them.